src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/SupplementaryEvidenceMapper.java

```diff
@@ -4,14 +4,15 @@
 import uk.gov.hmcts.reform.bulkscan.orchestrator.model.ccd.SupplementaryEvidence;
 import uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.model.Envelope;
 
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.model.ccd.mappers.DocumentMapper.mapDocuments;
+
 @Component
-public class SupplementaryEvidenceMapper implements ModelMapper<SupplementaryEvidence> {
+public class SupplementaryEvidenceMapper {
 
     public SupplementaryEvidenceMapper() {
         // empty mapper construct
     }
 
-    @Override
     public SupplementaryEvidence mapEnvelope(Envelope envelope) {
         return new SupplementaryEvidence(mapDocuments(envelope.documents));
```

Comment:
As you can it only needs the documents.

After this PR is merge and interface is gone, I'll change this method to something like
```
SupplementaryEvidence map(List<Document> existingDocument, List<Document> envelopeDocuments)
```